experimental/packages/opentelemetry-sdk-node/README.md

```diff
@@ -128,6 +128,10 @@ Configure a trace exporter. If an exporter OR span processor is not configured,
 
 Configure tracing parameters. These are the same trace parameters used to [configure a tracer](../../../packages/opentelemetry-sdk-trace-base/src/types.ts#L71).
 
+### resourceAttributes
+
+Configure [resource attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md), key names are in CONSTANT_CASE, e.g. `SERVICE_NAME`.
```

Comment:
I think we should mention that this works only for resource keys part of `@opentelemetry/semantic-conventions`.
And we should state how to use non standard resoure keys.